lib/Dialect/TritonGPU/Transforms/WarpSpecialization/AutomaticWarpSpecialization.cpp

```diff
@@ -36,7 +36,11 @@ void AutomaticWarpSpecialization::runOnOperation() {
   OpPassManager pm;
   pm.addPass(createTritonGPUPartitionScheduling());
   pm.addPass(createNVWSInsertAref());
+#if 0
```

Comment:
it is for convenience `#if .. #else .. #endif`, it basically allows me to swap from one to another, while debugging few remaining issues, but if you prefer it out, just let me know.